bukkit/src/main/java/com/leonardobishop/quests/bukkit/tasktype/type/PlaytimeTaskType.java

```diff
@@ -1,5 +1,6 @@
 package com.leonardobishop.quests.bukkit.tasktype.type;
 
+import com.earth2me.essentials.Essentials;
```

Comment:
This will cause issues if Essentials isn't present on the server (Essentials is only a softdepend of Quests, not a full dependency)